### ISSUE TITLE
Are the motors numbered differently in the CPP simulator drone from the Python example drone?

### ISSUE THREAD

**Issue by ZmeiGorynych** [udacity/FCND-Controls-CPP#10]
I can only make the controls work by assuming the motors in the C++ drone are numbered like [[1,2],[3,4]], whereas the motors in the Lesson 4 - 3D Drone Full Notebook.ipyb are quite clearly numbered as [[1,2],[4,3]].

Once I make that assumption, my solution works like a charm, producing graphs for drone attitude control consistent with the README - and if I assume the numbering as in the notebook, nothing works. 

Specifically, that impacts the `GenerateMotorCommands` function, where with the former numbering the last two lines look something like 
```
cmd.desiredThrustsN[2] = (c_bar + p_bar - q_bar - r_bar) / 4.f;
cmd.desiredThrustsN[3] = (c_bar - p_bar - q_bar + r_bar) / 4.f;
```
(note the signs) and work fine, whereas in line 137 of solution.py used in the last lesson it says
`omega_4 = (c_bar + p_bar - r_bar - q_bar)/4`

If my understanding is correct and the numbering is indeed different, it's worth highlighting it very prominently in the README to save other people the hours of debugging I had to do to figure this out ;)

**Comment by willhunt** [comment]
Yes, I found this confusing as well. A diagram would be useful. The answer is in the starter code for [QuadCOntrol.cpp](https://github.com/udacity/FCND-Controls-CPP/blob/master/src/QuadControl.cpp):

```
73|   cmd.desiredThrustsN[0] = mass * 9.81f / 4.f; // front left
74|   cmd.desiredThrustsN[1] = mass * 9.81f / 4.f; // front right
75|   cmd.desiredThrustsN[2] = mass * 9.81f / 4.f; // rear left
76|   cmd.desiredThrustsN[3] = mass * 9.81f / 4.f; // rear right
```
